src/utils/kubernetes.ts

```diff
@@ -112,16 +115,17 @@ export class Kubernetes implements KubernetesApi {
    }
 
    private async waitManifests(
-      manifestsPath: string,
-      namespace: string
+      namespace: string,
+      applicationName: string
    ): Promise<Errorable<string>> {
-      const command = `wait --for=condition=Ready -f "${manifestsPath}" -R --namespace ${namespace}`;
+      const command = `rollout status deployment/${applicationName} --namespace ${namespace} --timeout ${WAIT_TIMEOUT}`;
```

Comment:
along the same lines as my last comment, if we do want to eventually expand beyond just deployments (which I'm not sure if we do and if we don't then you can ignore this) do we want to keep a manifest-based readiness check and check separately for deployments/pods with the rollout check? 

I suppose if the readiness/condition check is outdated practice anyway then maybe there's no use for keeping that check in the first place? Just a thought

Reply:
upon further research it looks like the readiness check just applies to pod/pod-like resources anyway so I guess there's no point in keeping the readiness check, but wonder if it's worth it to keep the rollout check type-agnostic (so for this case feed in a type as a parameter) in case we want to start supporting other resource types in the future like statefulsets/replicasets? Again just a thought

Reply:
Swapped it around to support this